src/amd/compiler/aco_opcodes_cpp.py

```diff
@@ -61,6 +61,15 @@
 };
 
 }
+
+#if defined(DEBUG) || !defined(NDEBUG)
+const char *aco_opcode_string[(size_t) aco_opcode::num_opcodes] = {
+% for name in opcode_names:
+   "${name}",
+% endfor
+#endif
+};
+
```

Comment:
Not sure about this one. It completely disables the possibilty to print our IR in release mode.

Reply:
Yep, I didn't notice that when I wrote this patch.